Source/Scene/Model.js

```diff
@@ -306,6 +306,16 @@ define([
         this._releaseGltfJson = defaultValue(options.releaseGltfJson, false);
         this._animationIds = undefined;
 
+        // Undocumented options
```

Comment:
I can help you set up some test data for when we merge this into the 3d-tiles branch and need to confirm it still works after fixing the merge conflicts.